### PR TITLE
update ubuntu and go versions

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
update docker files:
- golang version to `1.17.6`
- ubuntu to `22.04`
- run update & upgrade

### CODE REVIEW

File: utils/elrond-go-keygenerator [hunk 1]
```diff
@@ -10,7 +10,8 @@ WORKDIR /go/elrond-go/cmd/keygenerator
 RUN go build
 
 # ===== SECOND STAGE ======
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```
Comment (AdoAdoAdo) [inline utils/elrond-go-keygenerator:13]: is there a limitation to use go 1.14.9?
I would change to 1.17.6 as well

Comment (liviuf-ancas) [reply utils/elrond-go-keygenerator:13]: why not bump it to go 1.19.3 which is the latest version of golang image?

Comment (bogdan-rosianu) [reply utils/elrond-go-keygenerator:13]: used 1.17.6, since it's currently the go version we support everywhere

**Review by AdoAdoAdo — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Comment by miiu96** [comment]
This PR is no longer needed.

**Pull request closed without merging**